components/ImageBanner/src/stories/bem.stories.mdx

```diff
@@ -0,0 +1,430 @@
+import { Canvas, Description, Meta, Story } from "@storybook/addon-docs";
+import { DesignTokenDocBlock } from "storybook-design-token/dist/doc-blocks";
+import pkg from "../../package.json";
+import readme from "../../README.md";
+import "../index.scss";
+
+<Meta
+  title="CSS/Actions/Image Banner"
+  parameters={{
+    componentSubtitle: `${pkg.name} - ${pkg.version}`,
+    status: {
+      type: "WORK IN PROGRESS",
```

Comment:
Moet deze status geupdate worden als hij approved is?

Reply:
Wat mij betreft wanneer het approved is door de denhaatg team, maar storybook nlds is natuurlijk een geval apart, kun je beter aan Yolijn vragen omdat zij 1 van de beheerders is van nlds storybook.